active/package-maintainers.md

```diff
@@ -0,0 +1,125 @@
+- GitHub issues and pull requests in a new `django/package-maintainers-wg` repository.
+- Collaborative documents set up in the Django Software Foundation’s Google Workspace.
+- A new private #package-maintainers-wg` channel on the Django Discord server for internal discussions.
+
+The group will also offer office hours for package maintainers every month via a video call.
+
+## Reporting
+
+The group will share highlights from current activities on a monthly basis via a forum post.
+
+## Appendix
+
+### Group activities
+
+As an illustration of the group’s remit, here are possible activities members could take part in.
+
+#### Support Django version compatibility efforts
+
+As Django regularly releases new versions, it’s an ongoing effort to ensure packages are compatible with new releases.
+
+Group members can support maintainers in this effort by:
+
+- Creating a shared calendar with Django release dates and deadlines.
+- Creating communication channels for package maintainers to coordinate compatibility changes for a specific release.
+- Recommending appropriate automation, such as [django-upgrade](https://github.com/adamchainz/django-upgrade) or how to set up Django pre-releases in Continuous Integration.
+
+#### Share packaging best practices
+
+The group can work on create, maintain, or curate best practices for Django package maintainers. For example, the official [Advanced tutorial: How to write reusable apps](https://docs.djangoproject.com/en/5.0/intro/reusable-apps/), or packaging-focused sections on third-party resources like [awesome-django](https://github.com/wsvincent/awesome-django).
+
+Those best practices can cover aspects like:
+
+- Documentation approaches and tools
+- Use of the `django_` prefix for app names (see [Change reusable apps naming recommendation](https://forum.djangoproject.com/t/change-reusable-apps-naming-recommendation/25233)).
+- Recommended Python, Django, browser, operating system support policies to align with those of Django itself.
+- Tooling, for example how to set up Continuous Integration for Django packages.
+
+#### Facilitate community-driven maintenance
+
+Group members can liaise with organizations dedicated to sharing maintenance efforts, such as [Jazzband](https://jazzband.co/), [Wagtail Nest](https://github.com/wagtail-nest), or [Django Commons](https://github.com/django-commons). Or work directly with packages in need of new maintainers to find new candidates. This can involve:
+
+- Sharing calls for new package maintainers
+- Coordinating trusted volunteer open source “roadies” familiar with package ownership transfers.
+- Facilitating participation in mentoring programs such as [Google Summer of Code](https://summerofcode.withgoogle.com/) or [Djangonaut Space](https://djangonaut.space/).
+- Providing support for tasks requiring very specific expertise, like funding avenues or vulnerability reports handling.
+
+#### Review the package ecosystem
```

Comment:
That's certainly an interesting section. Establishing metrics is hard, and even harder once people start to game them. Maybe something to consider (or not).

Reply:
I worry a bit that this could be seen as a way to give some of the "obvious" Django packages even more visibility. This has more to do with the fact that the packages I'm using most of the time are a bit more niche than they maybe should be so I'm a bit wary. Just as an example, counting releases or contributors doesn't necessarily say a lot when different packages are in the same space but do not have the same design goals (e.g. django CMS vs. django-content-editor)

I think there's a lot of value in curating packages and making lists. https://djangopackages.org/ exists already, and I'm sure that the packaging WG could contribute in some way to the same space. I wouldn't want you to remove this, but I do have a few concerns. Maybe joining the group would be a good way to help steer the boat... :-)

Reply:
I think choice of metrics would be important to avoid the feedback loop of popularity. I don't normally care about how many people are using a package as much as I do about the flow of PRs/commits/merges as a straw main for whether a package is maintained and the frequency of releases to understand how frequently I should expect to have to set aside time to do upgrades.

Reply:
Yeah, I'd be interested in metrics mostly as a way to avoid the popularity feedback loop. Without naming names I can think of a few packages I use that have <20 stars but that are better designed, keep up with Django updates better, and have better/more extensive tests than alternatives that have >1000 stars. Stars and other popularity metrics are mostly used as a proxy for the actual metrics people care about and in some cases are actually increasing the technical debt of the ecosystem.